sci-biology/jellyfish/jellyfish-1.1.11.ebuild

```diff
@@ -24,8 +24,8 @@ DEPEND="${RDEPEND}
 
 src_install(){
 	default
-	sed -e "s#jellyfish-${PV}#jellyfish#" -i "${D}"/usr/lib64/pkgconfig/jellyfish-1.1.pc || die
-	mkdir -p "${D}/usr/include/${PN}" || die
-	mv "${D}"/usr/include/"${P}"/"${PN}"/* "${D}/usr/include/${PN}/" || die
-	rm -rf "${D}/usr/include/${P}"
+	sed -e "s#jellyfish-${PV}#jellyfish#" -i "${ED}"/usr/lib/pkgconfig/jellyfish-1.1.pc || die
```

Comment:
Neither `lib` or `lib64` is good. use `$(get_libdir)`. It comes for free in `EAPI=6`, `inherit multilib` otherwise.